web/beacon-app/src/features/topics/components/TopicQuery.tsx

```diff
@@ -18,14 +24,17 @@ const TopicQuery = () => {
       </button>
 
       {open && (
-        <div className="flex space-x-1">
-          <p>
-            <Trans>
-              Coming soon! Query the topic for insights with EnSQL e.g. the latest event or last 10
-              events.
-            </Trans>
-          </p>
-        </div>
+        <>
+          <div className="flex space-x-1">
+            <p>
+              <Trans>
+                Coming soon! Query the topic for insights with EnSQL e.g. the latest event or last
```

Comment:
There's a separate story to update the copy. Once we have the link to the EnSQL documentation, I'll make the change.